content/blog/creating-dscc-block-storage-resources-using-dscc-ansible-playbooks.md

```diff
@@ -0,0 +1,192 @@
+
+```yaml
+    - name: Create GreenLake DSCC Volume Set
+      greenlake_volumeset:
+        config: "{{ config }}"
+        device_type: 1
+        system_id: "{{system_id}}"
+        state: present
+        data:
+          # app_set_business_unit: "HPE"
+          # app_set_comments: "Edit"
+          app_set_importance: "MEDIUM"
+          app_set_name: "{{volume_set_name}}"
+          # name: "ansible_volume_set_1"
+          app_set_type: "{{workload}}"
+          # members: ["ansible-vol1", "ansible-vol2"]
+
+    - debug: var=volume_sets
+```
+
+**Creation of Volume (Primera)**
+
+The mandatory fields are the name, size, and user CPG. Other parameters are optional.
+
+```yaml
+    - name: Create GreenLake DSCC volume
+      greenlake_volume:
+        config: "{{ config }}"
+        system_id: "{{system_id}}"
+        state: present
+        data:
+          comments: "Ansible library test"
+          count: 2 #Optional
+          data_reduction: True #Optional
+          name: "{{volume_name}}"  
+          size_mib: 16384
+          snap_cpg: "SSD_r6" #Optional
+          snapshot_alloc_warning: 10 #Optional
+          user_alloc_warning: 10 #Optional
+          user_cpg: "SSD_r6"
+
+    - debug: var=volumes
+```
+
+**Updating the volume set**
+
+The volume set will be updated with the volumes created earlier. For this, provide the id of the volume set and the names of the volumes created. Since these are created dynamically during the playbook execution, these are referred to as variables.
+
+```yaml
+    - name: Update GreenLake DSCC Volume Set with volumes
+      greenlake_volumeset:
+        config: "{{ config }}"
+        device_type: 1
+        system_id: "{{system_id}}"
+        state: present
+        data:
+          id: "{{volume_sets.0.id}}"
+          add_members:
+            - "{{volume_name}}.0"
+            - "{{volume_name}}.1"
+
+    - debug: var=volume_sets
+```
+
+**Exporting the volume set to the Host Group**
+
+Now, the volume set created needs to be exported to the host group created earlier. Provide the system id, host group id, and the volume set id in the form of variables.
+
+```yaml
+    - name: Export GreenLake DSCC Volume Set from system
+      greenlake_volumeset:
+        config: "{{ config }}"
+        device_type: 1
+        system_id: "{{system_id}}"
+        state: export
+        data:
+          id: "{{volume_sets.0.id}}"
+          host_group_ids:
+            - "{{host_groups.0.id}}"
+
+    - debug: var=volume_sets
+```
+
+I hope you found this blog post helpful. This is just a sample use case one can implement to create resources on Data Services Cloud Console. Admins\/users can come up with multiple use cases that can be used to manage the Data Services Cloud Console resources, like cleaning the resources, monitoring the resources, etc. Keep an eye out for new blogs and videos that are on track to be released related to the automation of Data Services Cloud Console operations.
```

Comment:
Here, I would add "HPE GreenLake" prior to the first mention of "Data Services Cloud Console." We often  call out the full name of a product in the summary.